src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/scriptSettings.properties

```
@@ -29,7 +29,7 @@ uploadtext=Select a Groovy script from your local system to be uploaded (*.groov
 UploadEncoding=Uploaded files should be encoded with UTF-8.
 Permission = Allow usage in build step
 Restriction = Restriction
-RestrictionDescription = Script is always executed on Master
+RestrictionDescription = Script is always executed on the controllerMaster

```

I think you meant to get rid of the term "master" here?
```suggestion
RestrictionDescription = Script is always executed on the controller
```